addons/mail/static/src/xml/text_emojis.xml

```diff
@@ -1,18 +1,29 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
-    <div t-name="mail.EmojisDropdown" t-attf-class="o_mail_emojis_dropdown o_mail_add_emoji dropdown">
+    <div t-name="mail.EmojisDropdown" class="o_mail_emojis_dropdown o_mail_add_emoji dropdown">
         <button class="btn btn-block dropdown-toggle px-3 py-1"
-            type="button" t-att-id="emojisDropdownId"
+            type="button"
             data-toggle="dropdown"
             aria-haspopup="true"
             aria-expanded="false"
             title="Add an emoji">
-            <i class="fa fa-smile-o"></i>
+            <i class="fa fa-smile-o"/>
         </button>
-        <div class="dropdown-menu dropdown-menu-right border-0 p-2" t-att-aria-labelledby="emojisDropdownId" style="width: 406px">
-            <t t-call="mail.Composer.emojis">
-                <t t-set="emojis" t-value="widget.emojis"></t>
-            </t>
+        <div class="dropdown-menu dropdown-menu-right border-0 p-2" style="width: 406px">
+            <t t-call="mail.emojisPicker"/>
         </div>
     </div>
+
+    <div t-name="mail.emojisPicker" class="o_mail_emoji_container">
+        <t t-foreach="widget.emojis" t-as="emoji">
+            <span
+                class="o_mail_emoji"
+                t-att-data-emoji="emoji.sources[0]"
+                t-att-data-emoji-unicode="emoji.unicode"
+                t-att-title="emoji.description"
+                t-att-aria-label="emoji.description">
+                <t t-raw="emoji.unicode"/>
+            </span>
+        </t>
+    </div>
```

Comment:
Why copy/paste the `mail.Composer.emojis ` template here instead of t-calling it?
Is it just to avoid `$target.text().trim()` in `_onEmojiClick` and use the `t-att-data-emoji-unicode` ?

Seems overkill.

Reply:
@awa-odoo because in the future we want to improve the emojis widget (have multiple tabs with different emojis, etc). So it's easier if this template belong to the emojis widget

Reply:
(and it also simplify the javascript code)

Reply:
Meh.
Not a big fan to do that "for the future".
It's not clear in this commit purpose.

I think we will split when the time comes, but the time has not come yet (and trust me, it may not come in the near future ^^).

Reply:
@awa-odoo ok, I'll just call the original template :)